### PR TITLE
fixed spelling and such

### PR DESCRIPTION
**Pull request by TallTed**
Question: Does W3C mandate `dd monthname yyyy` date format? I strongly suggest changing all such to ISO standard date format `YYYY-MM-DD`.

### CODE REVIEW

File: charter/index.html [hunk 1]
```diff
@@ -163,7 +163,7 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
...
+        <p>Cloud services in use on the Web today (2023) often require users to store their data and place control over that data at a third-party cloud provider. Solid adds to existing Web standards to enable user control: <q cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">to realize a space where individuals can maintain their autonomy, control their data and privacy, and choose applications and services to fulfill their needs.</q> Solid defines the notion of storage (one feature of the construct informally referred to as Solid Pods), in which users place their own data and control access to that data, and a suite of interoperable protocols for managing storage, applications that use storage, and interactions with existing protocols for authentication.
```
Comment (csarven) [inline charter/index.html:166]: This is a quote. Not sure how technical you'd like but wouldn't the following be more accurate (also noting now that you changed the spelling in an earlier PR):

```suggestion
        <p>Cloud services in use on the Web today (2023) often require users to store their data and place control over that data at a third-party cloud provider. Solid adds to existing Web standards to enable user control: <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">to [<span lang="en-US">realize</span>] a space where individuals can maintain their autonomy, control their data and privacy, and choose applications and services to [<span lang="en-US">fulfill<span>] their needs.</q> Solid defines the notion of storage (one feature of the construct informally referred to as Solid Pods), in which users place their own data and control access to that data, and a suite of interoperable protocols for managing storage, applications that use storage, and interactions with existing protocols for authentication.
```

**Review by csarven — APPROVED** [review]

**Comment by csarven** [comment]
>Does W3C mandate dd monthname yyyy date format?

It is in the charter template... I doubt it is mandated per se but it is one of those awkward date formats, with variations of day month year matching the date format in pubrules.

>I strongly suggest changing all such to ISO standard date format YYYY-MM-DD.

I agree. @pchampin , can we do that and then ask for forgiveness if anyone comes after us?

File: charter/index.html (hunk 1 above)
Comment (TallTed) [reply charter/index.html:166]: Oh, I hadn't noticed/realized that was a quotation block ... nor any others that may have been. I'm OK with falling back to GB spelling within such. And I should try to remember that Solid seems to be bilingually US/GB and somewhat prefer GB.

Comment (csarven) [reply charter/index.html:166]: Sure, that works too. The document is in en-US. Updated suggestion to keep lang en-GB.

File: charter/index.html [hunk 2]
```diff
@@ -240,7 +240,7 @@ <h3>
...
+                            The Solid Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way. <q cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralized Web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
```
Comment (TallTed) [inline charter/index.html:243]:
```suggestion
                            The Solid Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way. < q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralised Web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
```

**Review by csarven — APPROVED** [review]

File: charter/index.html [hunk 3]
```diff
@@ -240,7 +240,7 @@ <h3>
...
+                            The Solid Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way. < q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralised Web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
```
Comment (csarven) [inline charter/index.html:243]:
```suggestion
                            The Solid Protocol specification aims to provide applications with secure and permissioned access to externally stored data in an interoperable way. <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralised Web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
```

**Review by elf-pavlik — APPROVED** [review]

**Comment by TallTed** [comment]
Updated the date placeholders. Ready to merge, from my side.

**Review by pchampin — APPROVED** [review]

**Pull request merged**